### PR TITLE
fix: TT-139  Reduce the left margin on customers page

### PR DESCRIPTION
**Pull request by jr-98**
Margin left is done. Additionally I corrected the problem of resposivity in the header of the customer table.

### CODE REVIEW

File: src/app/modules/customer-management/components/customer-info/components/customer-list/customer-list.component.html [hunk 1]
```diff
@@ -1,23 +1,23 @@
...
   datatable
   [dtTrigger]="dtTrigger"
   [dtOptions]="dtOptions"
 >
   <thead class="thead-blue">
     <tr class="d-flex">
-      <th class="col-4">Customer ID</th>
-      <th class="col-5">Name</th>
-      <th class="col-3 text-center">Options</th>
+      <th class="col-sm-4">Customer ID</th>
```
Comment (scastillo-jp) [inline src/app/modules/customer-management/components/customer-info/components/customer-list/customer-list.component.html:10]: you can use only `col` to create responsive columns

Comment (jr-98) [reply src/app/modules/customer-management/components/customer-info/components/customer-list/customer-list.component.html:10]: no, becouse the colums of customers page has a specific size

File: src/app/modules/customer-management/components/customer-info/components/customer-list/customer-list.component.html [hunk 2]
```diff
@@ -1,23 +1,23 @@
...
   </thead>
   <app-loading-bar *ngIf="(isLoading$ | async)"></app-loading-bar>
   <tbody *ngIf="((isLoading$ | async) === false)">
     <tr class="d-flex" *ngFor="let customer of customers">
-      <td class="col-sm-4">{{ customer.id }}</td>
-      <td class="col-sm-5">{{ customer.name }}</td>
-      <td class="col-sm-3 text-center">
+      <td class="col col-sm-4">{{ customer.id }}</td>
+      <td class="col col-sm-5">{{ customer.name }}</td>
+      <td class="col col-sm-3 text-center">
```
[the remark this answers is not in the document]
Comment (jr-98) [reply src/app/modules/customer-management/components/customer-info/components/customer-list/customer-list.component.html:20]: OK I going to review. Thaks

File: src/app/modules/customer-management/components/customer-info/components/customer-list/customer-list.component.html [hunk 3]
```diff
@@ -1,23 +1,23 @@
 <table
   *ngIf="customers"
-  class="table table-sm table-bordered table-striped mb-0"
+  class="table table-responsive-sm table-bordered table-striped mb-0"
```
Comment (scastillo-jp) [inline src/app/modules/customer-management/components/customer-info/components/customer-list/customer-list.component.html]: Responsive tables allow tables to be scrolled horizontally with ease. Make any table responsive across all viewports by wrapping a .table with .table-responsive. Or, pick a maximum breakpoint with which to have a responsive table up to by using .table-responsive{-sm|-md|-lg|-xl}.

example:

```
<div class="table-responsive">
  <table class="table">
    ...
  </table>
</div>
```

If you need use table-responsive-sm, the  correct form is; 

```
<div class="table-responsive-sm">
  <table class="table">
    ...
  </table>
</div>
```

[Documentation link](https://getbootstrap.com/docs/4.1/content/tables/)

File: src/app/modules/customer-management/components/customer-info/components/customer-list/customer-list.component.scss [hunk 4]
```diff
@@ -0,0 +1,4 @@
+.col-3 .col-4 .col-5 {
```
Comment (Angeluz-07) [inline src/app/modules/customer-management/components/customer-info/components/customer-list/customer-list.component.scss:1]: You are overriding boostrap classes. I think it's better create a new class.

**Review by Angeluz-07 — COMMENTED** [review]
You may need to rebase with master. So tests can pass and also to have your branch updated.

File: src/app/modules/customer-management/components/customer-info/components/customer-list/customer-list.component.html [hunk 5]
```diff
@@ -1,23 +1,24 @@
-<table
+<div class="table table-responsive">
+  <table
   *ngIf="customers"
-  class="table table-sm table-bordered table-striped mb-0"
+class="table table-bordered table-striped"
```
Comment (Angeluz-07) [inline src/app/modules/customer-management/components/customer-info/components/customer-list/customer-list.component.html:4]: Maybe we  can add the first whitespace back.

File: src/app/modules/customer-management/components/customer-info/components/customer-list/customer-list.component.scss (hunk 4 above)
Comment (jr-98) [reply src/app/modules/customer-management/components/customer-info/components/customer-list/customer-list.component.scss:1]: Can you give me more information?

Comment (Angeluz-07) [reply src/app/modules/customer-management/components/customer-info/components/customer-list/customer-list.component.scss:1]: It is ok. As far as I know, there are bootstrap classes named as `col-3` `col-4` `col-5`. So, I think is not a good idea to re-define them, that could cause unexpected behaviours. 

It is better to define a new class. e.g :
```
.myNewClass {
  padding: 1rem 0rem 2rem;
  overflow: hidden;
}
```

File: src/app/modules/customer-management/components/customer-info/components/customer-list/customer-list.component.html [hunk 6]
```diff
@@ -38,6 +39,7 @@
     </tr>
   </tbody>
 </table>
```
Comment (scastillo-jp) [inline src/app/modules/customer-management/components/customer-info/components/customer-list/customer-list.component.html:41]: indentation

File: src/app/modules/customer-management/components/customer-info/components/customer-list/customer-list.component.scss (hunk 4 above)
Comment (Angeluz-07) [reply src/app/modules/customer-management/components/customer-info/components/customer-list/customer-list.component.scss:1]: If I am wrong somehow, please let me know.

**Comment by juanultimate** [comment]
Not sure why you have three commits for this PR. IIMHO, one commit is enough, or at least you can squash these two commits
image

**Comment by josepato87** [comment]
> Not sure why you have three commits for this PR. IIMHO, one commit is enough, or at least you can squash these two commits
> image

@juanultimate He asked me about those commits, I suggested to leave them as it, but when merging we would squash this PR. Besides, I suggested to put better names in his commits, because it's useful to see the whole history of changes when reviewing the code.

**Review by juanultimate — APPROVED** [review]

**Review by scastillo-jp — APPROVED** [review]

**Pull request merged**